packages/lisk-p2p/src/p2p.ts

```diff
@@ -215,12 +215,14 @@ export class P2P extends EventEmitter {
 					const peerId = constructPeerId(socket.remoteAddress, wsPort);
 
 					const incomingPeerInfo: P2PDiscoveredPeerInfo = {
-						...queryObject,
 						ipAddress: socket.remoteAddress,
 						wsPort,
 						height: queryObject.height ? +queryObject.height : 0,
+						isDiscoveredPeer: true,
```

Comment:
is this `true`? it sounds like if it's incoming connection, it should be `false`?

Reply:
This is correct because the `isDiscoveredPeer` property of a `P2PDiscoveredPeerInfo` should always be true. It allows us to distinguish complete from incomplete peer info at runtime.

Reply:
maybe change the name in other issue to make it clear?